ui/src/services/NodeVolumesUtils.js

```diff
@@ -79,28 +78,28 @@ export const computeVolumeGlobalStatus = (name, status) => {
   const condStatus = condition?.status;
   const condReason = condition?.reason;
 
-  switch(condStatus) {
-  case 'True':
-    return STATUS_AVAILABLE;
-  case 'False':
-    return STATUS_FAILED;
-  case 'Unknown':
-    switch(condReason) {
-    case 'Pending':
-      return STATUS_PENDING;
-    case 'Terminating':
-      return STATUS_TERMINATING;
+  switch (condStatus) {
+    case 'True':
+      return STATUS_READY;
```

Comment:
You've also changed `AVAILABLE` to `READY` in this commit: it's not only about indentation.

I think you need to extract this part.

Reply:
You are right! But I couldn't split the big switch into two hunks and commit them separately...
Let's keep it in this way then :)